index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+
+<body>
+  <header class="hello">
+
+    <H1>
+      Hello microverse
+    </H1>
```

Comment:
- [OPTIONAL] HTML tag should always be in lowercase:notebook: . Please avoid using uppercase for your tag. Can you please update `H1` with `h1`